build.sbt

```diff
@@ -147,7 +147,7 @@ lazy val laws = crossProject(JSPlatform, JVMPlatform).in(file("laws"))
  * (such as IOApp). This is the "batteries included" dependency.
  */
 lazy val core = crossProject(JSPlatform, JVMPlatform).in(file("core"))
-  .dependsOn(kernel, laws % Test, testkit % Test)
+  .dependsOn(kernel, concurrent % Test, laws % Test, testkit % Test)
```

Comment:
Should depend on **concurrent** directly, rather than just in `% Test` scope. **core** is meant to be the "batteries included" distributable, so it should fold everything together except the benchmarks and the examples.